integration/test_utils.go

```diff
@@ -295,6 +298,15 @@ func KillProcess(name string) error {
 	return nil
 }
 
+// KillPid kills the process by pid. kill is used.
+func KillPid(pid int) error {
+	output, err := exec.Command("kill", "-9", strconv.Itoa(pid)).CombinedOutput()
```

Comment:
i think you wanted to send a SIGTERM here (-15) not a SIGKILL? or maybe the KillProcess is wrong as I see it's using default which I believe is SIGTERM.

Reply:
Use kill directly, which uses SIGTERM by default.

Reply:
:-) good